aperturedb/PyTorchDataset.py

```diff
@@ -76,9 +77,9 @@ def __init__(self, db, query, label_prop=None):
             batch = r[self.find_image_idx]["FindImage"]["batch"]
             self.total_elements = batch["total_elements"]
         except:
-            print("Query error:")
-            print(self.query)
-            print(self.db.get_last_response_str())
+            logger.error("Query error:")
```

Comment:
if making this a logger.error should this be multiple calls or just one?

Reply:
done